app/controllers/users/transfers_controller.rb

```diff
@@ -23,12 +29,16 @@ def destroy
     private
 
     def transfer_params
-      transfer_params = params.require(:dossier_transfer).permit(:email, :dossiers)
-      if transfer_params[:dossiers].present?
-        transfer_params.merge(dossiers: [current_user.dossiers.find(transfer_params[:dossiers])])
+      transfer_params = params.require(:dossier_transfer).permit(:email, :dossier)
```

Comment:
improve param naming (which confused me) : form send a single `dossier`, not a array